modules/log6x-enabling-loki-alerts.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // observability/logging/logging-6.0/log6x-loki.adoc
+// observability/logging/logging-6.2/log6x-loki-6.2.adoc
```

Comment:
Typically it's 

// * 

in our docs. I see that some examples have this asterisk and others do not.